docs/getting-started/openshift/index.md

```diff
@@ -0,0 +1,180 @@
+# ArgoCD Agent Hub and Spoke Cluster Setup Documentation Using Operator
+This document outlines the process of setting up ArgoCD Agent in a hub and spoke cluster architecture. This configuration allows a central ArgoCD instance (the hub) to manage applications deployed across multiple remote Kubernetes clusters (the spokes) through the argocd-agent.
+## Prerequisites
+Before proceeding with the setup, ensure you have the following:
+
+A running openshift/kubernetes cluster designated as the hub, with ArgoCD/openshift GitOps Operator installed.
+One or more Kubernetes clusters designated as spokes, ArgoCD/openshift GitOps Operator installed.
+oc configured to access both hub and spoke clusters.
+Operator must be installed in cluster scope mode.
+Apps in any Namespace should be enabled.
+Argocd-agentctl binary (for non-production scenario)
+
+
+
+## Hub Cluster Setup
+
+Hub cluster will be installed in argocd namespace
+
+Before you install the principal's manifest, you will need to create three
```

Comment:
This sentence seems incomplete. Should we mention Principal component before this? Its referred to as "hub" in general in this document.
Also, would be nice to have this in bullets

Reply:
"Principal" and "hub" are two very different entities. Please refer to https://argocd-agent.readthedocs.io/latest/concepts/components-terminology/